funcx_endpoint/funcx_endpoint/executors/high_throughput/funcx_manager.py

```diff
@@ -487,8 +490,14 @@ def remove_worker_init(self, worker_type):
 
         logger.debug("[WORKER_REMOVE] Appending KILL message to worker queue {}".format(worker_type))
         self.worker_map.to_die_count[worker_type] += 1
-        self.task_queues[worker_type].put({"task_id": pickle.dumps(b"KILL"),
-                                           "buffer": b'KILL'})
+        task_id = "KILL"
+        cont_id = "RAW"
+        task_buf = "KILL"
+        raw_buf = f'{task_id};{cont_id};{task_buf}'
```

Comment:
You could probably use `task.pack()` to construct the buffer here?

Reply:
I think we cannot use `Task::pack()` here, because we are trying to construct a `Task` variable, which will be put on the `task_queues`.

Reply:
I think that should work? something like this should work? probably even no need to define like lines 493-495
```
task = Task(task_id='KILL',
            cont_id='RAW',
            task_buf='KILL')
task.pack()
```

Reply:
Yes, I agree.